File: backend/src/main/java/com/backend/ord/domain/dto/BanksUsedInGameDTO.java
package com.backend.ord.domain.dto;

import com.backend.ord.domain.dto.abstracts.DTOBase;
import lombok.*;

import java.util.UUID;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class BanksUsedInGameDTO {
    private UUID id;

    private GameDTO game;
    private BankDTO bank;
}

File: backend/src/main/kotlin/com/backend/ord/domain/persistance/dto/BanksUsedInGameDTO.kt
package com.backend.ord.domain.persistance.dto

import com.backend.ord.domain.persistance.dto.game.GameDTOBase
import java.util.*

data class BanksUsedInGameDTO(
    val id: UUID = UUID.randomUUID(),

    val game: GameDTOBase<*>,
    val bank: BankDTO
)
